### PR TITLE
docs: Add draft getting started page

### PR DESCRIPTION
**Pull request by lukeblevins**
GitHub Issue (If applicable): Discussed internally N/A

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?
No docs for getting started

## What is the new behavior?
Added docs for getting started

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable): N/A

### CODE REVIEW

File: doc/How-Tos/GettingStarted/UsingUnoExtensions.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- *MyProjectName.Tests* and *MyProjectName.UI.Tests* - for writing unit and UI tests respectively.
+
+![The structure of the generated solution](../../images/ProjectStructure-min.png)
+
+
+# Running the Application 
+
+1. Select a target from the drop-down as pictured above
+
+TODO: Update screenshot so that the solution/project names are "MyProjectName"
```
Comment (nickrandolph) [inline doc/How-Tos/GettingStarted/UsingUnoExtensions.md:49]: @lukeblevins can you address this TODO please

**Comment by nickrandolph** [comment]
@agneszitte-nventive / @Xiaoy312 do you mind taking a quick read through this getting started doc and add any comments on things that aren't clear

**Comment by agneszitte-nventive** [comment]
> @agneszitte-nventive / @Xiaoy312 do you mind taking a quick read through this getting started doc and add any comments on things that aren't clear

tagging @dr1rrb also

**Comment by lukeblevins** [comment]
Also adding @jeromelaban to lay eyes on this one, as per the request of FT

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**